exporter/jaegerexporter/exporter.go

```diff
@@ -100,3 +126,52 @@ func (s *protoGRPCSender) pushTraceData(
 
 	return 0, nil
 }
+
+func (s *protoGRPCSender) shutdown(context.Context) error {
+	close(s.stopCh)
+	s.stopWg.Wait()
```

Comment:
I think there is a race with lines 148-149. Lines 131-123 can execute and shutdown returns while execution is between lines 148 and 149. It means shutdown will not be effective.

Reply:
True, I think it would have been mostly innocuous, but I changed it in name of correctness.